model_compression_toolkit/common/post_training_quantization.py

```diff
@@ -371,25 +366,27 @@ def _prepare_model_for_quantization(in_model: Any,
                                  representative_data_gen)  # model reading
     graph.set_fw_info(fw_info)
 
-
     if tb_w is not None:
         tb_w.add_graph(graph, 'initial_graph')
 
     ######################################
-    # Graph substitution (pre statistics collection)
+    # Graph substitution (prepare graph)
     ######################################
-    transformed_graph = substitute(graph, fw_impl.get_substitutions_pre_statistics_collection())
-
-    if tb_w is not None:
-        tb_w.add_graph(transformed_graph, 'pre_statistics_collection_substitutions')
+    graph = substitute(graph, fw_impl.get_substitutions_prepare_graph())
 
     #########################################
     # Set prior info to nodes
     ##########################################
-    for node in transformed_graph.nodes:
+    for node in graph.nodes:
         node.prior_info = fw_impl.get_node_prior_info(node=node,
                                                       fw_info=fw_info)
```

Comment:
Why does the batch-normalization information is not initialized in get_node_prior_info?